pkg/template/template.gen.go

```diff
@@ -0,0 +1,78 @@
+// Copyright 2016 Istio Authors
```

Comment:
As we just discussed offline, stuff that is authored by non-Mixer developer should not be in the //pkg/* directory. So let's move the template protos to the top-level //template directory.